lib/Fmt_ast.ml

```diff
@@ -3554,7 +3480,7 @@ and fmt_functor_arg c {loc; txt= arg} =
 
 and fmt_module_type c ({ast= mty; _} as xmty) =
   let ctx = Mty mty in
-  let {pmty_desc; pmty_loc; pmty_attributes} = mty in
+  let {pmty_desc; pmty_loc; pmty_attributes; pmty_ext= _} = mty in
```

Comment:
Why is it ok to ignore pmty_ext here ?

Reply:
It should not have been added to module_type since no extensions are allowed, I will remove it.